src/pages/index.tsx

```diff
@@ -32,383 +38,157 @@ import CreateWalletContent from "!!raw-loader!../data/CreateWallet.js"
-  height: 100%;
-  width: 100%;
-  margin: 0;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    padding: 2rem;
-  }
-`
-
-const LeftColumnContent = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-`
-
-const IntroLeftColumn = styled(LeftColumn)`
-  padding: 6rem;
-  height: 100%;
-  width: 100%;
-  margin: 0;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    padding: 2rem;
-  }
-  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
-    padding: 0rem;
-  }
-`
-
-const StyledIcon = styled(Icon)`
-  fill: ${(props) => props.theme.colors.text};
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-  }
-  &:hover {
-    fill: ${(props) => props.theme.colors.primary};
-  }
-  &:active {
-    fill: ${(props) => props.theme.colors.primary};
-  }
-  &:focus {
-    fill: ${(props) => props.theme.colors.primary};
-  }
-`
-
-const H2 = styled.h2`
-  margin: 0 0 1.5rem;
-`
-
-const StyledH2 = styled.h2`
-  margin-bottom: 0.5rem;
-  font-family: sans-serif;
-  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
-    font-size: 1.5rem;
-  }
-`
-
-const StyledCardList = styled(TitleCardList)`
-  margin-left: 4rem;
-  max-width: 624px;
-  border: 1px solid ${(props) => props.theme.colors.text};
-  box-shadow: ${(props) => props.theme.colors.cardBoxShadow};
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin-left: 0rem;
-    max-width: 100%;
-  }
-`
-
-const StyledCalloutBanner = styled(CalloutBanner)`
-  margin: 8rem 0 4rem;
-  padding: 2rem 4rem;
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    margin-bottom: 4rem;
-    padding: 2rem;
-  }
-`
+type ChildOnlyProp = { children: ReactNode }
+
+const SectionHeading = (props: HeadingProps) => (
+  <Heading
+    lineHeight={1.4}
+    fontFamily="sans-serif"
+    fontSize={{ base: "2xl", sm: "2rem" }}
+    fontWeight={600}
+    mb={2}
+    {...props}
+  />
+)
+
+const SectionDecription = (props: ChildOnlyProp) => (
+  <Box mb={8} fontSize={{ base: "md", sm: "xl" }} lineHeight={1.4} {...props} />
+)
+
+const ImageContainer = (props: FlexProps & { children: ReactNode }) => (
+  <Flex width={{ base: "75%", lg: "full" }} height="full" {...props} />
+)
+
+const CardContainer = (props: {
+  children: ReactNode
+  minChildWidth: SimpleGridProps["minChildWidth"]
+}) => (
+  <Flex
+    flexWrap="wrap"
+    gap={8}
+    p={{ lg: 4 }}
+    width="full"
+    sx={{
+      "& > *": {
+        minW: props.minChildWidth,
+      },
+    }}
+  >
+    {props.children}
+  </Flex>
+)
+
+const ContentBox = (props: ChildOnlyProp) => (
+  <Box py={4} px={{ base: 4, lg: 8 }} {...props} />
+)
+
+const HomeActionCard = (props: {
+  cardItem: Record<"title" | "description" | "alt" | "to", string> & {
+    image: IGatsbyImageData | undefined
+  }
+  cardBoxShadow: string
+}) => (
+  <Box
+    as={ActionCard}
```

Comment:
`ActionCard` has not migrated yet, so having to resort to this override implementation to be able to accept incoming Chakra Props.